### ISSUE TITLE
ALSA warnings at startup

### ISSUE THREAD

**Issue by diegodeodati** [jasperproject/jasper-client#667]
Hi,
when i start jasper.py everytime i got a lot of warnings about ALSA.
There is a way to abolish them?

example:
ALSA lib confmisc.c:1286:(snd_func_refer) Unable to find definition 'cards.bcm2835.pcm.front.0:CARD=0'
ALSA lib conf.c:4259:(_snd_config_evaluate) function snd_func_refer returned error: No such file or directory
ALSA lib conf.c:4738:(snd_config_expand) Evaluate error: No such file or directory
ALSA lib pcm.c:2239:(snd_pcm_open_noupdate) Unknown PCM front
ALSA lib pcm.c:2239:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.rear
ALSA lib pcm.c:2239:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.center_lfe
ALSA lib pcm.c:2239:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.side

**Comment by G10DRAS** [comment]
Yes there is way but not simple. I did something 2 years back and don't remember what I did right now.

**Comment by diegodeodati** [comment]
Could you please check it? And if is possible send me the resolution steps?

Il 04 mar 2018 06:02, "G10DRAS" <notifications@github.com> ha scritto:

**Comment by G10DRAS** [comment]
ok check this out
https://groups.google.com/forum/#!topic/robocopai/UnbVwW8sOGo
